content/geoip/docs/web-services/_schemas/ResponseRepresentedCountry.mdx

```diff
@@ -10,12 +10,18 @@ import responseJson from '../_examples/geoip';
   users of the IP address. For instance, the country represented by an overseas
   military base.
 
+  [Learn about represented countries on our Knowledge
```

Comment:
more is missing.

Reply:
yes, in this case i wanted "learn" instead of "learn more," because i don't think it's very clear what a registered or represented country is at all from the API schema